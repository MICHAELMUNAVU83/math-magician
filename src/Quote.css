@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@200;300&family=Josefin+Sans:wght@100&family=Kanit:ital,wght@1,300;1,400&family=Kdam+Thmor+Pro&display=swap');
h3{
    display: flex;
    justify-content: center;
    align-items: center;
    width: 40%;
    font-family: 'Kanit', sans-serif;
    margin-left: 25%;
    font-size: 30px;
    margin-top: 15%;
    height: 100%;
}