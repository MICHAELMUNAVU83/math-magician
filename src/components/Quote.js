import React from 'react';

function Quote() {
  return (
    <div>
      <h3>
        Without mathematics, there’s nothing you can do. Everything around you
        is mathematics. Everything around you is numbers. — Shakuntala Devi,
        Indian writer and mental calculator
      </h3>
    </div>
  );
}

export default Quote;
