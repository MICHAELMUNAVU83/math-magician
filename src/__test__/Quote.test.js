import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

it('Qoute renders correctly', () => {
  const quotetree = renderer
    .create(<Quote />)
    .toJSON();
  expect(quotetree).toMatchSnapshot();
});
