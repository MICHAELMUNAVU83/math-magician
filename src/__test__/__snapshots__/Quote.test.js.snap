// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Qoute renders correctly 1`] = `
<div>
  <h3>
    Without mathematics, thereâ€™s nothing you can do. Everything around you is mathematics. Everything around you is numbers.
  </h3>
</div>
`;
